Dockerfile

```diff
@@ -15,11 +15,11 @@ COPY . .
 RUN make build-ethermint-linux
 
 # Final image
-FROM alpine:edge
+FROM golang:1.14 as final
 
-# Install ca-certificates
-RUN apk add --update ca-certificates
-WORKDIR /root
+WORKDIR /
+
+RUN apt-get update
```

Comment:
correct me if I'm wrong but this change should be the only one made in order to make this work for all the OS?

Reply:
well all this does is allow the binaries created from the previous build step (build-env) to be executable on the final image